tools/ci_build/github/windows/eager/requirements.txt

```diff
@@ -2,5 +2,4 @@ setuptools
 wheel
 numpy
 typing_extensions
-torch
-
+torch==1.11.0
```

Comment:
Did you intentionally remove the empty line at the end of the file? Sometimes the empty line is intentional / desired...

Reply:
my local file shows the line so strange it is not here. auto-format removes extras when i save. does not appear to be causing an issue so resolving this.

Reply:
Looking at the original file in my local copy, it looks like it has _2_ trailing blank lines! (and this would correctly normalize it to a single blank line..)